.github/kubernetes-compute/tool.sh

```diff
@@ -2,8 +2,9 @@
 set -x
 
 # Global variables
-export LOCK_FILE=$0.lock
-export RESULT_FILE=amlarc-test-result.txt
+export SCRIPT_DIR=$( cd  "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+export LOCK_FILE=${SCRIPT_DIR}/$0.lock
+export RESULT_FILE=${SCRIPT_DIR}/amlarc-test-result.txt
```

Comment:
do we need to rename the result file name?

Reply:
changed